people_tool/templates/people_tool/partials/courses.html

```diff
@@ -6,14 +6,46 @@ <h3 class="breadcrumbs">
     {% verbatim %}
     <small><i class="fa fa-chevron-right"></i></small>
     <a href="#/">Search People</a>
-    <small><i class="fa fa-chevron-right"></i></small>
-    {{ personId }}
+
+    <span ng-if="selectedPersonInfo.name_first">
+        <small><i class="fa fa-chevron-right"></i></small>
+        {{selectedPersonInfo.name_last}},&nbsp;{{selectedPersonInfo.name_first}}'s Courses
+    </span>
     <span ng-if="operationInProgress"><i class="fa fa-refresh fa-spin"></i></span>
   </h3>
 </nav>
 
 <main>
+ <div ng-if='selectedPersonInfo' class="row">
+
+  <div class="col-md-12">
+      <div id="personInfoDT_wrapper" class="dataTables_wrapper form-inline no-footer">
 
+        <table id="personInfoDT"
+               class="table table-striped table-hover ng-isolate-scope dataTable no-footer person-info-table"
```

Comment:
Do we need the `.ng-isolate-scope`? And is the `.no-footer` appropriate both on this table element and the wrapper div above this?